Sources/TUSKit/TUSClient.swift

```diff
@@ -6,8 +6,8 @@ import Foundation
 import BackgroundTasks
 import UIKit
 
+
 /// Implement this delegate to receive updates from the TUSClient
-@available(iOS 13.4, macOS 10.13, *)
```

Comment:
This should be updated to iOS 15 right?

Reply:
The react-native app is going to be updated to ios 15 min so I think it'll be okay to remove

Reply:
And the podspec defines min as 15 in tuskit
  ```s.ios.deployment_target = '15.0'```